RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -186,6 +200,18 @@ private extension ManageSubscriptionsViewModel {
             } catch {
                 self.state = .error(error)
             }
+        case .customUrl:
```

Comment:
Note this is only currently available in the `ManageSubscriptionView`. It won't be available in the `NoSubscriptionsView` or `WrongPlatformView` since we're not using the paths for those views.

Reply:
interesting... It might be worth figuring out a way for those to work eventually since for those it might make even more sense - you can direct users to where you'd actually want them to manage

Reply:
Yeah, I think we need to move some of the logic to a shared view for all 3 of them and handle paths in all 3. Right now the buttons are mostly hardcoded in those 2 other views. Probably not something for this PR though.